Resources/Prototypes/_CM14/Roles/Jobs/Civilians/survivor.yml

```diff
@@ -0,0 +1,46 @@
+﻿- type: job
+  id: Survivor
+  parent: JobBaseCM
+  name: job-name-survivor
+  description: job-description-survivor
+  playTimeTracker: JobSurvivor
+  requirements:
+  - !type:RoleTimeRequirement
+    role: JobRifleman
+    time: 18000 #5 hours
+  - !type:RoleTimeRequirement
+    role: JobHospitalCorpsman
+    time: 18000 #5 hours
+  - !type:RoleTimeRequirement
+    role: JobCombatTech
+    time: 18000 #5 hours
+  startingGear: SurvivorGear
+  icon: "JobIconSurvivor"
+  requireAdminNotify: false
+  joinNotifyCrew: false
+  supervisors: job-supervisors-command # no one probably?
```

Comment:
Wiki says they respond to nobody but themselves
They have no idea who the marines are or their intentions

Reply:
done.